app/scripts/controllers/network.js

```diff
@@ -51,6 +51,10 @@ module.exports = class NetworkController extends EventEmitter {
   }
 
   lookupNetwork () {
+    // Prevent firing when provider is not defined.
+    if (!this.ethQuery || !this.ethQuery.sendAsync) {
+      return
```

Comment:
can we add a `console.warn` here so that we see this happening if something goes wrong outside of tests